server/src/test/java/com/mycompany/app/AppTest.java

```diff
@@ -0,0 +1,20 @@
+package com.mycompany.app;
```

Comment:
path for tests should be

```javatest/com/google/speech/tools/voxetta/...```

This isn't actually testing anything (is it used to establish a maven test rule?) If it's unused, it should be removed

Reply:
was just a file crated by the scaffold, will delete

Reply:
also added the folder structure for that